README.Rmd

```diff
@@ -16,14 +16,16 @@ knitr::opts_chunk$set(
 )
 ```
 
-# substrait
+# Substrait R Package for Generating Cross-language Specification of Data Compute Operations
```

Comment:
This is a lot of words for a title. I think it might be better to go for something simpler like *Substrait R Interface* in the title. The explanation of what Substrait is already included in the text below.

Reply:
```suggestion
# Substrait R Interface
```

Reply:
Working at the same time on this @ianmcook. Title change is done.